ci/docker/bosh-oracle-cpi-boshrelease/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:15.04
+MAINTAINER CF Dev <cloud_dev_cf_ww_grp@oracle.com>
+
+# Update base image
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+RUN locale-gen en_US.UTF-8; dpkg-reconfigure locales
+RUN apt-get update; apt-get -y upgrade; apt-get clean
+
+# Install dependencies
+RUN apt-get -y install build-essential git-core curl wget jq sudo; apt-get clean
+RUN apt-get install -y zip unzip; apt-get clean
+RUN apt-get install -y sqlite3 libsqlite3-dev; apt-get clean
+RUN apt-get install -y mysql-client libmysqlclient-dev; apt-get clean
+RUN apt-get install -y libpq-dev; apt-get clean
+RUN apt-get install -y python python-setuptools libyaml-dev libpython-dev; apt-get clean
+RUN apt-get install -y ruby vim; apt-get clean
+RUN apt-get remove python-pip
+RUN easy_install pip Jinja2
+
+# Install Golang
+ENV GOLANG_VERSION 1.8.1
```

Comment:
We've been using 1.8.3.  We should bump it here too

Reply:
ok